plugins/include/amxmodx.inc

```diff
@@ -156,16 +156,26 @@ forward client_connect(id);
 forward client_authorized(id);
 
 /**
- * Called when a client is disconnecting from the server.
+ * @deprecated This function does not catch all cases.
```

Comment:
deprecation does not make the documentation invalid, we haven't removed it in all other cases either